src/core/src/core_logic/PatchInstaller.py

```diff
@@ -800,4 +810,20 @@ def get_max_batch_size(self, maintenance_window, package_manager):
         self.composite_logger.log_debug("Calculated max batch size is: {0}".format(max_batch_size_for_packages))
 
         return max_batch_size_for_packages
-
+    
+    def is_install_patches_in_single_batch_enabled(self):
+        """"""
+        if self.install_patches_in_single_batch == True:
+            return self.install_patches_in_single_batch
+        
+        dur = datetime.datetime.strptime(self.execution_config.duration, "%H:%M:%S")
+        dura = timedelta(hours=dur.hour, minutes=dur.minute, seconds=dur.second)
```

Comment:
possible to rename this variable it's too generic, make it more descriptive on what it is trying to do, maybe something like patch_duration?